From 9e1227a1f3d0bd53a9321154a1769162cd161276 Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.de>
Date: Tue, 18 Jun 2013 12:48:58 +0200
Subject: [PATCH] Don't switch camera on HP EB 8780
References: bnc#797090
Patch-Mainline: Never (kABI issue)

Some cameras on HP EB 8780 don't work at SS
Just don't switch unconditionally

Signed-off-by: Oliver Neukum <oneukum@suse.de>
---
 drivers/usb/host/pci-quirks.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/usb/host/pci-quirks.c b/drivers/usb/host/pci-quirks.c
index 4aaf1b9..74ef935 100644
--- a/drivers/usb/host/pci-quirks.c
+++ b/drivers/usb/host/pci-quirks.c
@@ -578,6 +578,13 @@ static const struct dmi_system_id bt_excepted_from_switch_table[] = {
 		},
 		.driver_data = (void *)64,
 	},
+        {
+                /* HP EliteBook 8780 */
+                .matches = {
+                        DMI_MATCH(DMI_PRODUCT_NAME, "HP EliteBook 8780w"),
+                },
+                .driver_data = (void *)64,
+        },
 	{ }
 };
 
-- 
1.8.1.4

